bodhi/validators.py

```diff
@@ -148,7 +148,7 @@ def validate_build_tags(request):
                 if not build_rel:
                     raise KeyError("Couldn't find release from build tags")
             except KeyError:
-                msg = 'Cannot find release associated with build: {}, tags: {}'.format(build, tags)
+                msg = 'Cannot find any tags associated with that nvr: {}, tags: {}'.format(build, tags)
```

Comment:
`tags` in this context are what the `build` is tagged with, so that message wouldn't be correct.

Also, I'm not sure that `nvr` is the most user-friendly way to describe a `build`, even though most people probably know what it means.